src/ansys/fluent/core/solver/flobject.py

```diff
@@ -1169,6 +1240,24 @@ def allowed_values(self):
             return []
 
 
+class _InputFilePurposeMixin:
+    def file_purpose(self):
```

Comment:
I don't think we need this function, we can simply check `isinstance(argument, _InputFilePurposeMixin)`.

Reply:
Yes, removed. Thank you.